lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java

```diff
@@ -192,18 +191,56 @@ private LambdaUpdateEventConfigurationTaskOutput updateEventConfiguration(Lambda
 
     private LambdaEventConfigurationDescription formEventObject(String curr, LambdaUpdateEventConfigurationTaskInput taskInput) {
         LambdaEventConfigurationDescription singleEvent = LambdaEventConfigurationDescription.builder().eventSourceArn(curr).batchsize(taskInput.getBatchsize()).enabled(true)
+                .maxBatchingWindowSecs(taskInput.getMaxBatchingWindowSecs())
                 .account(taskInput.getAccount()).credentials(taskInput.getCredentials()).appName(taskInput.getAppName())
                 .region(taskInput.getRegion()).functionName(taskInput.getFunctionName()).qualifier(taskInput.getQualifier()).
                         build();
         if (curr.startsWith(DYNAMO_EVENT_PREFIX) || curr.startsWith(KINESIS_EVENT_PREFIX)) {
             if (StringUtils.isNullOrEmpty(taskInput.getStartingPosition())) {
                 taskInput.setStartingPosition(DEFAULT_STARTING_POSITION);
             }
+
+            singleEvent.setBisectBatchOnError(taskInput.getBisectBatchOnError());
```

Comment:
Looks good. Did you test these when the testInput does not have these parameters. Does it handle nulls correctly? ARe you sure you dont need the equivalent of StringUtils.isNullOrEmpty around these?